src/components/NavBar.vue

```diff
@@ -7,8 +7,16 @@
         </div>
       </router-link>
       <div class="login-container">
-        <!-- <i class="pi pi-bars menu-icon" @click="toggleMenu" /> -->
-        <!-- <PvMenu ref="menu" id="overlay_menu" :model="dropdownItems" :popup="true" /> -->
+        <div v-if="isSuperAdmin">
```

Comment:
I still think that we need to remove this div because we want partner admins to see the menu as well.

Reply:
changed!